src/Common/src/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -55,7 +55,7 @@ protected Marshaller()
         /// <returns>The  created Marshaller</returns>
         public static Marshaller CreateMarshaller(TypeDesc parameterType, PInvokeMethodData pInvokeMethodData, ParameterMetadata pInvokeParameterdata)
         {
-            MarshallerKind marshallerKind = GetMarshallerKind(parameterType, pInvokeParameterdata.MarshalAsDescriptor, pInvokeMethodData, pInvokeParameterdata.Return);
```

Comment:
Nit: These lines are very long - can't even see the end in the github diff UI. Could you please break them?